psake.ps1

```diff
@@ -102,15 +104,30 @@ Task Build {
     }
 }
 
-Task Test -depends Build {
+Task Test {
```

Comment:
Since the Test task is used in the `ShowBuildMenu.bat` tool, building all frameworks when trying to test only one is excessive.  The `Build` task has been added to the task list of each of the CI scripts.